README.md

```diff
@@ -5,38 +5,43 @@
 
 > Rust & RIOT combined for ergonomic embedded development
 
-This is an experimental project to provide a nice base OS for embedded
-development on low-end IoT devices (with some kilobytes of RAM/flash, think
-Cortex-M). It combines the awesome Rust embedded ecosystem with
-[RIOT](https://github.com/RIOT-OS/RIOT).
+RIOT-rs is a project aiming to provide a general-purpose
+operating system adequate for cybersecure, memory-safe, low-power IoT.
 
-**This is highly experimental. Expect heavy changes and breakage!**
+RIOT-rs is based on Rust from the ground up, and uses formal verification for critical modules. 
+For a more verbose rant, see this
+[manifesto](https://future-proof-iot.github.io/RIOT-rs/dev/manifesto.html).
 
-If you're looking for a more production ready way of writing RIOT applications
-in Rust, check out [riot-wrappers](https://gitlab.com/etonomy/riot-wrappers).
+Overall, RIOT-rs aims for a 'batteries-included' experience, on par
+with [RIOT](https://github.com/RIOT-OS/RIOT): 
+all the needed OS facilities and a high level of integration for 
+the various libs, tools and toolchains required by low-power IoT application
+development. Targets include various hardware based on 
+32-bit microcontroller architectures (such as Cortex-M, RISC-V).
+In particular, RIOT-rs aims for:
 
-## Goals
-
-- improve RIOT using the merits of Rust.
-- provide a "rusty" development workflow (e.g., using cargo / crates.io)
-- provide a nice Rust API, framework and collection of crates suitable for
-  embedded development
-- rewrite parts of RIOT in Rust to improve robustness and maintainability
+- **application code portability** across all supported hardware, via consistent memory/energy efficient APIs;
+- programming based either/both on **async & preemptive scheduling** paradigms, the latter using formally verified modules using [hax](https://hacspec.org/blog/posts/hax-v0-1/);
```

Comment:
would it make sense to split this in two, async first (mentioning that it is based on embassy) and second "preemptive scheduling", (using verified modules based on hax)?

Reply:
good suggestion. Now split.